common/primitives/src/lib.rs

```diff
@@ -6,7 +6,6 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 // Strong Documentation Lints
 #![deny(
-	missing_docs,
```

Comment:
Did we all agree to remove this?

Reply:
it was a duplicate -- the rule is still at the bottom of the list.